### DISCUSSION TITLE
How to customize animation properties easily

### DISCUSSION THREAD

**Discussion by webpixeldesign** [Codeinwp/otter-blocks#1451] [Q&A]
How can i customize animation properties like transform % etc easily?
For now I tried to achieve this by editing the blocks-animation/build/animation/index.css file using the Plugin File Editor option in the WordPress, but I'm guess all my customization will get wiped everytime i update the plugin.

I don't know how to use wp hooks, or even if this is possible to do via hooks. Can anyone help me?

I want to do this to achieve subtle animations, where the elements transform only 10% and change opacity.

**Comment by Soare-Robert-Daniel** [comment]
Hi, 

You can use custom CSS to modify the animation at the block level.

1. Enable the feature in the Otter dashboard via `Settings > Otter` in the Admin panel.

![image](https://user-images.githubusercontent.com/17597852/213379685-a1d6fcf6-1468-4a36-8c8e-be8eacf1ba06.png)

2. On each block, you will have Block Tools; click on the plus and enable Custom CSS.

![image](https://user-images.githubusercontent.com/17597852/213379918-f47930c5-ffc4-4b32-94ef-fff5e3e14190.png)

3. You can write CSS code like this:

![image](https://user-images.githubusercontent.com/17597852/213380475-6215eb1d-eaa1-4464-acee-0d4cc9e8b13b.png)

ℹ️ `selector` is a unique keyword from us in which we refer to the targeted block.

**Reply by webpixeldesign** [reply]
i want to change default properties for all animations actually, and not on per block basis. also i'm using the blocks animation plugin and not otter, sorry forgot to mentioned that in op discussion message

**Reply by Soare-Robert-Daniel** [reply]
You can use something like [this](https://www.wpbeginner.com/plugins/how-to-easily-add-custom-css-to-your-wordpress-site/l) and target our CSS animation classes from `blocks-animation/build/animation/index.css`

**Reply by webpixeldesign** [reply]
ok so i should write my own keyframes and then use them in target custom css,
